tests/test_metrics.py

```diff
@@ -0,0 +1,18 @@
+import time
+import unittest
+
+from judoscale.core.metric import Metric
+
+
+class TestMetricsForWeb(unittest.TestCase):
+    def test_heroku_metrics(self):
+        # Heroku timestamp is in integer milliseconds
+        heroku_timestamp = str(round(time.time() * 1000))
+        metric = Metric.for_web(heroku_timestamp)
+        self.assertAlmostEqual(metric.value, 0, delta=1)
+
+    def test_nginx_metrics(self):
+        # Nginx timestamp is in float seconds, in the format "t=123.456"
+        nginx_timestamp = str(f"t={time.time()}")
+        metric = Metric.for_web(nginx_timestamp)
+        self.assertAlmostEqual(metric.value, 0, delta=1)
```

Comment:
Nice!! 💯

Reply:
I've tweaked the tests for Metric slightly — I added a tiny `sleep` into both tests (10ms each) to assert verify that latency calculation does take place correctly.

I've been running `watch -n 0.1 'python -m unittest discover -s tests'` in the background for hours and so far those parameters (10ms with 5ms delta) do not yield any false positives. But this is dependent on the load on the system, so if it ever becomes an issue in CI I think we'll just loosen the bounds a little bit.

Reply:
Tweaked the parameters slightly, because there were a couple of random failures with 10ms and 5ms delta.